CMakeLists.txt

```diff
@@ -115,3 +115,8 @@ configure_file(templates/setup.sh.in ${CMAKE_CURRENT_BINARY_DIR}/setup.sh @ONLY)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/setup.sh
     DESTINATION ${CMAKE_INSTALL_PREFIX}
 )
+
+# copying test scripts
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/scripts/subdetector_tests
+  DESTINATION ${CMAKE_INSTALL_PREFIX}/scripts
```

Comment:
```suggestion
install(PROGRAMS scripts/subdetector_tests/g4_material_scan.py
  DESTINATION ${CMAKE_INSTALL_BINDIR}
```

Reply:
I think we could put this script in `bin` instead of `scripts/subdetector_tests` instead of installing it as a program